include/tvm/runtime/memory/memory_manager.h

```diff
@@ -37,15 +37,15 @@
 
 namespace tvm {
 namespace runtime {
-namespace vm {
+namespace memory {
 
 struct Buffer {
   /*! \brief The pointer to the allocated block of memory. */
   void* data{nullptr};
   /*! \brief The size of the block. */
   size_t size{0};
   /*! \brief The shape of the tensor. */
-  std::vector<int64_t> shape;
+  ShapeTuple shape;
```

Comment:
we can remove this member `ShapeTuple shape;`, and related `buf.shape.push_back(shape[i]);` in naive_allocator.h. Looks it is not used.

Reply:
Agreed. Even the two stage memory allocation for OpenCL texture doesn't need it.